### DISCUSSION TITLE
The initial start is messy

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-androidapp#3973] [Ideas]
The initial start is messy. Now we have:

- [ ] The classic Onboarding screen
- [ ] The Changelog
- [ ] The Anonymous Analytics consent screen

We need a way to make it visually less overwhelming:
- [ ] Inline clickable cards on the hope screen for the last two ?
- [ ] Integrate them within the onboarding screen ?
- [ ] Something else ?

https://user-images.githubusercontent.com/1689815/112747021-0920e200-8fb3-11eb-943a-0a0fb021ed20.mp4

**Comment by teolemon** [comment]
Related issue: https://github.com/openfoodfacts/openfoodfacts-androidapp/issues/3874

**Comment by vtanakov** [comment]
It is a good idea to integrate the Anonymous Analytics consent to onboarding flow, I vote for it.
About the changelog screen - maybe we should turn off automatic presenting and make a button for it? in burger menu or on home screen(it is empty anyway)

**Reply by teolemon** [reply]
yes, good idea

**Reply by naivekook** [reply]
@teolemon about changelog - is it ok if we always will show the link on home screen? 
or we can create more tricky logic like:
- show this link only for the new version
- show this link only 3 first time a user opens the app

<img src="https://user-images.githubusercontent.com/1501831/117258548-c249c600-ae55-11eb-9d2f-9bffd1b84ea5.png" width="259" height="533">

**Reply by naivekook** [reply]
@teolemon friendly reminder

**Comment by naivekook** [comment]
Also if we want to change onboarding I found out some TODOs in the code from @VaiTon, so we can combine these tasks and refactor this screen
```
* TODO: redesign it & change the content
* TODO: explain the 3 scores
* TODO: be honest about offline until we implement offline scan (nobody cares about offline edit)
* TODO: perhaps highlight ingredient analysis 
```

**Reply by naivekook** [reply]
small UI issue for Russian language
![Screenshot_1620138942](https://user-images.githubusercontent.com/1501831/117020667-55bcb300-acff-11eb-8021-7729435a13f4.png)

**Comment by VaiTon** [comment]
@teolemon we have a new onboarding screen. What remains to be done?
